docs/contributing-to-airbyte/README.md

```diff
@@ -35,11 +35,12 @@ Before jumping into the code please first:
 This will enable our team to make sure your contribution does not overlap with existing works and will comply with the design orientation we are currently heading the product toward.
 If you do not receive an update on the issue from our team, please ping us on [Slack](https://slack.airbyte.io)!
 
-#### 2. Let's code
+### 2. Let's code
 1. To contribute to a connector, fork the [Connector repository](https://github.com/airbytehq/airbyte). To contribute to the Airbyte platform, fork our [Platform repository](https://github.com/airbytehq/airbyte-platform).  
 2. Open a branch for your work.
 3. Code, and please write **tests**.
 4. Ensure all tests pass. For connectors, this includes acceptance tests as well. 
+5. For connectors, make sure to increment the connector's version according to our [Semantic Versioning](#semantic-versioning-for-connectors) guidelines.
```

Comment:
Not sure if this is the correct way to link to an anchor on the same page

Reply:
It is!

Reply:
sweet!